muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/communicator/body/application/message-view/message.tsx

```diff
@@ -277,7 +280,7 @@ class Message extends React.Component<MessageProps, MessageState> {
       this.props.message.userGroupRecipients.map(
         (ug): ContactRecipientType => ({
           type: "usergroup",
-          value: ug,
+          value: ug as any,
```

Comment:
Can the any be avoided?

Reply:
Not specifically in this branch thats why type any. It will be resolved following branches, when more usergroup related paths and interfaces are replaced with generated api.